docs/troubleshooting-case-study.md

```
@@ -0,0 +1,641 @@
+sudo mkswap /swapfile
+sudo swapon /swapfile
+free -h
+```
+
+Verification:
+
+```text
+Swap: 2.0Gi
+```
+
+### Mistake Encountered
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)